inc/Mailer.class.php

```diff
@@ -715,6 +716,14 @@ public function send() {
             if(trim($data['headers'])) {
                 $headers .= MAILHEADER_EOL.trim($data['headers']);
             }
+            if(!function_exists('mail')){
+                $emsg = $lang['email_fail'] . $subject;
+                error_log($emsg);
+                if(in_array('admin', $USERINFO['grps'])){
+                    msg($emsg);
+                }
```

Comment:
Sorry that I didn't remember to attach the `msg()` usage, but you could let msg itself decide who is admin, and also `admin` group could be non-admin (you could just use `MSG_MANAGERS_ONLY` here and it should allow admin + manager):

https://github.com/splitbrain/dokuwiki/blob/31a58aba4c24b34c34ad5764d1a35b7c398c3a2c/inc/infoutils.php#L328-L357